tasks/debian10.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Install Pacemaker cluster packages to all nodes
+  apt:
+    name: "{{ cluster_node_is_remote | bool | ternary(pacemaker_remote_packages, pacemaker_packages) }}"
+    state: 'present'
+
+- name: Check if Corosync configuration is default configaration
+  shell: '/usr/bin/dpkg --verify corosync'
+  register: result
+
+- name: Destroy default configuration
+  shell: "pcs cluster destroy"
```

Comment:
(medium) could you please edit this to use `pcs_cluster` module for destroying the cluster instead of `shell`?
~~~
  pcs_cluster:
    state: 'absent'
~~~

Reply:
Oh yes I didn't think of it but It clearly make sense

Reply:
good.